src/pages/send/confirm/confirm.ts

```diff
@@ -337,6 +336,7 @@ export class ConfirmPage extends WalletTabsChild {
 
     if (this.tx.paypro) this.paymentTimeControl(this.tx.paypro.expires);
 
+    const goBack = this.wallets.length > 1 ? false : true;
```

Comment:
`const goBack = this.wallets.length > 0`

Reply:
The idea here is that if you have more than 1 wallet, you will have the opportunity to choose another if you do not have enough to pay